how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -151,24 +129,54 @@
     "    capture_hardware_metrics=True,\n",
     "    capture_stderr=True,\n",
     "    capture_stdout=True,\n",
-    "    source_files=\"hello_neptune.ipynb\",  # does not work in Colab\n",
+    "    source_files=\"hello_neptune.ipynb\",  # Doesn't work in Colab\n",
     "    tags=[\"quickstart\", \"notebook\"],\n",
-    "    dependencies=\"requirements.txt\",  # pass the path to the requirements.txt file\n",
+    "    dependencies=\"requirements.txt\",  # Passes the path to the requirements.txt file\n",
```

Comment:
Is this common knowledge that the file has to exist first? For me it wasn't so I got an error and this step was skipped :D

Reply:
Yes.. that is why the comment says to "pass the path to the file".
The updated comment makes it sound like Neptune passes the path for you, which is not accurate.

Reply:
Yep, we can assume the audience knows about this concept. Environment info (in this case, Python packages and versions) is useful to capture as part of the overall experiment tracking.

However, the parameter is easy to overlook, and not everyone has a dependencies file specifically named `requirements.txt` in their project, so it probably shouldn't be the default option.

WDYT: should we use `"infer"`? Or require uncommenting the parameter?

Reply:
`infer` doesn't work for notebooks, hence we had decided to use `requirements.txt`. Not everyone uses this name, but most do.

We could comment this line out and have users uncomment it and customize it as per their needs, but I don't think that'll make much of a difference discoverability-wise. If someone skims through comments asking them to pass the path, they'll most probably skim through this as well without commenting it.

The error, in the current case, at least makes them aware that something is not done the way it should be.

I'll leave the phrasing of the comment to the tech writers :)

Reply:
> `infer` doesn't work for notebooks

Ahh, makes sense. I either hadn't registered that or I forgot about it.

I think I'd still favor having it commented, to keep it semi-discoverable but not get in people's way if passing `requirements.txt` won't work well for them. I'd assume more people run this in Colab than download the notebook, so this would remove some friction from the typical notebook quickstart experience.

Reply:
I don't think it gets in the way though. Neptune throws a non-exiting error if the file cannot be found.
What I think could be improved is throwing a warning instead of an error: https://github.com/neptune-ai/neptune-client/pull/1739

Reply:
I suppose we can leave it uncommented now, but I'd suggest rephrasing the comment to be more clear and helpful overall. Something like this maybe

```suggestion
    "    dependencies=\"requirements.txt\",  # You can pass the path to your dependencies file or omit this parameter\n",
```